CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.4.0 (WIP)
+  
+  * Added support for string keys in form fields and removed autoconversion of strings to atoms (#319)
```

Comment:
We actually didn't remove the autoconversion in general, right? It was more like a side-effect of changing the type to `:any`. So I believe something like this might be more useful to the user:

```
Support defining form fields as strings. Consequently, fields defined as literal strings will no 
longer be auto-converted to `:atom` and will keep the original value (#319)
```

**Note:** I think the changelog should be just the initial place to check for breaking changes, however, we should probably write a short migration guide giving some basic instructions on how to fix those breaking changes. That's usually not necessary but for `v0.4` might be a good idea as we're introducing a few of them.

@surface-ui/core WDYT?

Reply:
Your version is definitely better!